src/contrasts.jl

```diff
@@ -159,12 +159,12 @@ function ContrastsMatrix(contrasts::C, levels::AbstractVector{T}) where {C<:Abst
     # if levels are defined on contrasts, use those, validating that they line up.
     # what does that mean? either:
     #
-    # 1. contrasts.levels == levels (best case)
+    # 1. DataAPI.levels(contrasts) == levels (best case)
```

Comment:
```suggestion
    # 1. levels(contrasts) == levels (best case)
```

Reply:
same as above (although here it's just for clarifying the logic so doesn't need to run)